### PR TITLE
Remove old style scenarios

### PR DESCRIPTION
**Pull request by kretep**
Removes scenarios from the builder (gui and schema)

Pages now have only one editor for the content.

### CODE REVIEW

File: lib/ReactViews/RCBuilder/RCPageEditor/RCPageEditor.jsx [hunk 1]
```diff
@@ -128,10 +139,15 @@ function RCPageEditor(props) {
...
           >
+            <option value="INTRODUCTION">Introduction</option>
             <option value="SCOPE">Scope</option>
             <option value="HOTSPOTS">Remote climate</option>
             <option value="CONNECTION">Relevance to EU</option>
+            <option value="CLIMATE_EVENT">Climate event</option>
+            <option value="LOCAL_IMPACT">Local impact</option>
+            <option value="CONNECTION_IMPACT">Connection impact</option>
             <option value="EU_IMPACT">EU impact</option>
+            <option value="GLOBAL_IMPACT">Global impact</option>
```
Comment (sjvrijn) [inline lib/ReactViews/RCBuilder/RCPageEditor/RCPageEditor.jsx]: Do you know if these are actually used somewhere?
In https://github.com/RECEIPT-H2020/terriajs/commit/71c9326e449275c7b713bcc9d4e5cb5d4fcfbaa8 I recently added those options that were missing in the schema. As I was just trying to get it to work again, I didn't want to risk breaking anything by removing the (seemingly) unused ones or changing the order.

Do you think we can leave them out instead? (And maybe fix the enum in the schema at the same time 🙈 )

**Review by sjvrijn — COMMENTED** [review]
Apart from my question about the section options, this looks good to me 👍

File: lib/ReactViews/RCBuilder/RCPageEditor/RCPageEditor.jsx (hunk 1 above)
Comment (kretep) [reply lib/ReactViews/RCBuilder/RCPageEditor/RCPageEditor.jsx]: Good point; I had assumed the enum was up-to-date. Maybe @goord has a list of sections to use?

Comment (sjvrijn) [reply lib/ReactViews/RCBuilder/RCPageEditor/RCPageEditor.jsx]: My best guess is this from the navigation buttons:
https://github.com/RECEIPT-H2020/terriajs/blob/71c9326e449275c7b713bcc9d4e5cb5d4fcfbaa8/lib/ReactViews/NavigationBars/NavigationTopBar.jsx#L161-L178

Comment (kretep) [reply lib/ReactViews/RCBuilder/RCPageEditor/RCPageEditor.jsx]: I have updated the list and the schema according to the buttons!

**Review by sjvrijn — APPROVED** [review]
lgtm

**Pull request merged**